spk/gnupg/Makefile

```diff
@@ -9,7 +9,7 @@ MAINTAINER = SynoCommunity
 DESCRIPTION = GnuPG allows to encrypt and sign your data and communication, features a versatile key management system as well as access modules for all kinds of public key directories.
 STARTABLE = no
 DISPLAY_NAME = GnuPG
-CHANGELOG = "Initial release"
+CHANGELOG = "1. Initial release<br>2. Add pinentry dependency"
```

Comment:
Our changelog isn't cumulative, so this should probably be limited to `CHANGELOG = Add pinentry dependency`.
Can be done on merge.